exercises/supervised-classification/ex_rnw/ex_softmax.Rnw

```diff
@@ -1,22 +1,49 @@
-\begin{enumerate}
-  \item[a)] What is the relationship between softmax $\pi_k(x)=\frac{\exp(\theta_k^T x)}{\sum^g_{j=1} \exp(\theta_j^T x)}$ and the logistic function $\pix =\frac{1}{1+\exp(-\theta^{T} x)}$ for $g = 2$ (binary classification)?
+\begin{enumerate}[a)]
 
-  \item[b)] The likelihood function of a multinomially distributed target variable with $g$ target classes is given by
- $$\mathcal{L}_i = \P(Y^{(i)} = y^{(i)}|x^{(i)},\theta_1, \dots, \theta_g) = 
-\prod^g_{j=1} \pi_j(x^{(i)})^{\mathds{1}_{\{y^{(i)} = j\}}}$$
-where the posterior class probablities $\pi_1(x), \dots, \pi_g(x)$ are modeled with softmax regression.
-  Derive the likelihood function of $n$ such independent target variables. How can you transform this likelihood function into an empirical risk function?
-  \\ Hints: 
+  \item What is the relationship between softmax 
+  $$\pikx=\frac{\exp(\thetab_k^T \xv)}{\sumjg \exp(\thetab_j^T \xv)},
+  ~~ k = 1, 2$$
+  and the logistic function $$\pix = \frac{1}{1 + \exp(-\thetab^{T} \xv)}$$
+  for $g = 2$ (binary classification)?
+  
+  \item The likelihood function for a multinomially distributed target variable 
+  with $g$ target classes is given by\footnote{
+  While this might look somewhat complicated, it is actually just a 
+  very concise way to express the multinomial likelihood: for each observation, 
+  all factors but the one corresponding to the true class $j$ will be 1 (due to 
+  the 0 exponent), so the result is simply $\pi_j(\xi)$.
+  }
+  $$\LL_i(\thetab) = \P(\yi | \xi, \thetab_1, \thetab_2, \dots, \thetab_g) = 
+  \prod_{j = 1}^g \pi_j(\xi)^{\Ind{[\yi = j]}}$$
```

Comment:
Ist die Indikatorfunktion in latex-math nicht anders? Auf dem Cheatsheet ist es ein großes I, gefolgt von runden Klammern. Bzw. hatten wir alternativ glaube ich eckige Klammern ohne irgendwas davor?

Reply:
Das Symbol ist aus latex-math, aber mit den Klammern sind wir glaub nicht 100 % konsistent. Habs gegen runde ausgetauscht